frontend/javascripts/oxalis/view/left-border-tabs/layer_settings_tab.tsx

```diff
@@ -356,6 +381,7 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
     layerName: string,
     elementClass: string,
     layerSettings: DatasetLayerConfiguration,
+    draggingDisabled?: boolean,
```

Comment:
```suggestion
    draggingDisabled: boolean = true,
```

would make the condition in 472 clearer, I think.